src/abstractdataframe/abstractdataframe.jl

```diff
@@ -786,16 +786,17 @@ end
 ##
 ## Hashing
 ##
-## Make sure this agrees with isequals()
-##
 ##############################################################################
 
-function Base.hash(df::AbstractDataFrame)
-    h = hash(size(df)) + 1
+const hashdf_seed = UInt == UInt32 ? 0xfd8bb02e : 0x6215bada8c8c46de
```

Comment:
It'd be nice to use the general convention of all uppercase for constants.

Reply:
Actually I've followed what Base does with `AbstractArray` and in other places. I'd rather be consistent with that pattern.

Reply:
Okay, that's fine